qiskit_nature/second_q/mappers/qubit_converter.py

```diff
@@ -312,7 +265,7 @@ def force_match(
 
     def convert_match(
         self,
-        second_q_ops: SparseLabelOp | ListOrDictType[SparseLabelOp],
+        second_q_ops: SparseLabelOp | PauliSumOp | ListOrDictType[SparseLabelOp | PauliSumOp],
```

Comment:
Why can this now take a `PauliSumOp` as input?

Reply:
We can discuss that again: The idea is that the basic mode based mapping mapper.map() would act as the identity on Pauli Operator and as a mapper on Label operator. This would simplify the logic for aux_operators (which can be a dict of FermionicOp and QubitOperator) and also allow us to call map multiple times without it failing.

Reply:
Ah right I remember now. This is okay by me except for the aspect which I only thought of once commenting on `QubitMapper.map` below:
- we need to include `SparsePauliOp` in favor of `PauliSumOp` (maybe both for the time being? Not 100% sure right now)
- we need to update the docstrings to make this behavior more clear

Reply:
As discussed, I rolled back these changes